src/LearnositySdk/Utils/Json.php

```diff
@@ -104,4 +110,63 @@ public static function isJson($val)
     {
         return !is_null(json_decode($val, true));
     }
+
+    /**
+     * This function builds up a map of float values in scientific notation and the equivalent float
+     * notations. json_encode encodes float values in scientific notation, but we want to make sure
+     * they are in float notation
+     *
+     * @param $json the $json to be encoded
```

Comment:
Can you rename this variable to $value to keep it in sync with the method's signature?